src/cpu/core_dyn_x86/cache.h

```diff
@@ -355,10 +356,11 @@ class CodePageHandler : public PageHandler {
 	Bitu phys_page;
 };
 
-
-static INLINE void cache_addunsedblock(CacheBlock * block) {
-	block->cache.next=cache.block.free;
-	cache.block.free=block;
+static inline void cache_add_unused_block(CacheBlock *block)
+{
+	// block has become unused, add it to the freelist
+	block->cache.next = cache.block.free;
+	cache.block.free = block;
 }
```

Comment:
The new code is identical, but this is question:

If cache blocks are chained in a list, then shouldn't any other blocks that point to this incoming unused block also have their `block->cache.next` values set to `cache.block.free`?